portable/GCC/ARM_CA9/portASM.S

```diff
@@ -295,6 +296,10 @@ vApplicationIRQHandler:
 
 	LDR		r1, vApplicationFPUSafeIRQHandlerConst
 	BLX		r1
+	

```

Comment:
Again grateful if you can provide an explanation of this change.  What if the IRQ handler disables interrupt?